src/http/server.rs

```diff
@@ -787,6 +849,7 @@ impl<'a> EspHttpConnection<'a> {
         Ok(buf.len())
     }
 
+    // As per `EspHttpConnection::write`.
```

Comment:
?

Reply:
The `write` and `write_all` methods behave the same way, except for the return value. I would have expected `write_all` to flush the buffers or the like.

From my point of view, having both methods is a bit confusing.